src/DatePicker/DatePickerInput.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import DayPickerInput from 'react-day-picker/DayPickerInput';
+
+import Box from '../Box';
+import DatePicker from './DatePicker';
+import DatePickerNavbar from './DatePickerNavbar';
+import Input from '../Input';
+import PopoverContent from '../Popover/PopoverContent';
+
+const CustomOverlay = ({ classNames, selectedDay, children, ...props }) => (
+  <Box position="absolute">
+    <PopoverContent className={classNames.overlayWrapper} {...props}>
+      <DatePicker {...children.props} />
+    </PopoverContent>
+  </Box>
+);
+
+const DatePickerInput = ({ dayPickerProps, ...props }) => (
+  <DayPickerInput
+    component={Input}
+    overlayComponent={CustomOverlay}
+    dayPickerProps={{
+      navbarElement: DatePickerNavbar,
+      showOutsideDays: true,
+      weekdaysShort: ['S', 'M', 'T', 'W', 'T', 'F', 'S'],
```

Comment:
We're duplicating this and a couple other 'dayPickerProps' within our DayPicker usage. Is there a way to consolidate this? We have a constants file but that seems to global. Maybe constants within the `DayPicker` folder?

Reply:
I like that idea.